core/src/main/java/pl/agh/edu/actor/component/rating/Star.java

```diff
@@ -71,6 +71,7 @@ public void updateOverState(int rating) {
 		} else {
 			changeStateToDimmed();
 		}
+
```

Comment:
dlaczego?

Reply:
Wspominaliście o wysłaniu pustego commita żeby ponownie odpaliło testy i zastanawiałe się czy to zadzaiła